Fix the tests so they don't fail :)

exdir2 man2 transform: #159557 #162565
grab updates from newer automake

--- automake-1.8.5/tests/man2.test
+++ automake-1.8.5/tests/man2.test
@@ -41,7 +41,7 @@
 $AUTOMAKE
 
 # Let's play with $DESTDIR too, it shouldn't hurt.
-./configure --prefix=''
+./configure --prefix='' --mandir=/man
 DESTDIR="`pwd`/_inst" $MAKE -e install
 
 test -f ./_inst/man/man2/foo.2
--- automake-1.8.5/tests/transform.test
+++ automake-1.8.5/tests/transform.test
@@ -56,7 +56,7 @@
 $AUTOCONF
 $AUTOMAKE
 
-./configure --program-prefix=gnu- --prefix `pwd`/inst
+./configure --program-prefix=gnu- --prefix `pwd`/inst --mandir `pwd`/inst/man
 $MAKE
 $MAKE test-install
 $MAKE uninstall
--- automake-1.8.5/tests/exdir2.test
+++ automake-1.8.5/tests/exdir2.test
@@ -25,11 +25,11 @@
 set -e
 
 cat > Makefile.am << 'EOF'
-html_DATA = zot.html
+quuz_DATA = zot.html
 pkgdata_DATA = qbert
 EOF
 
 $ACLOCAL
 AUTOMAKE_fails
 grep 'pkgdatadir' stderr && exit 1
-grep 'Makefile.am:1:.*htmldir.*undefined' stderr
+grep 'Makefile.am:1:.*quuzdir.*undefined' stderr
